package.json

```diff
@@ -13,21 +18,25 @@
   "engines": {
     "node": ">= v0.8.0"
```

Comment:
Can you please bump this to >= v6. It's a shame we can't do platform specific versions given Windows now requires v8. I searched and the only solution is a failing install script.